index.html

```diff
@@ -5,10 +5,21 @@
     <title>All-Inclusive-Web-Team</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <nav>
-      <img id='logo' src="assets/dcrdlogo.png" alt="logo">
-    </nav>
+    <div id="nav-bar">
```

Comment:
As truongmleon@ noted, use `<nav>` for navigation containers. This helps convey your meaning more effectively ("this is a nav bar") and allows screen readers to take actions that are specific to nav bars.